src/components/table/OcTableFiles.vue

```diff
@@ -379,6 +371,12 @@ export default {
     areAllResourcesSelected() {
       return this.selection.length === this.resources.length
     },
+
+    selectedIds() {
+      let ids = []
```

Comment:
Computed prop can be solved as a one liner:
`return this.selectedResources.map(r => r.id)`

Reply:
yes that looks better